src/Tizen.NUI.Components/Controls/ItemsView/ItemsLayouter.cs

```diff
@@ -0,0 +1,281 @@
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        protected ItemsView ItemsView{get;set;}
+
+        /// <summary>
+        /// The last scrolled position which is calculated by ScrollableBase. The value should be updated in the Recycle() method.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected float PrevScrollPosition{get;set;}
+
+        /// <summary>
+        /// First index of visible items.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected int FirstVisible = -1;
+
+        /// <summary>
+        /// Last index of visible items.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        protected int LastVisible = -1;
+
+        /// <summary>
+        /// Visible ViewItem.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        protected List<ViewItem> VisibleItems = new List<ViewItem>();
+
+        /// <summary>
+        /// Flag of layouter initialization.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        protected bool IsInitialized = false;
+
+
+        /// <summary>
+        /// Candidate item step size for scroll size measure.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        protected float StepCandidate;
+
+        /// <summary>
+        /// Content size of scrollable.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        protected float ScrollContentSize;
+    
+        /// <summary>
+        /// boolean flag of scrollable horizontal direction.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        protected bool isHorizontal;
+
+        /// <summary>
+        /// Clean up ItemsLayouter.
+        /// </summary>
+        /// <param name="view"> ItemsView of layouter. </param>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual void Initialize(ItemsView view)
+        {
+            ItemsView = view;
+            Container = view.ContentContainer;            
```

Comment:
view의 null 확인 코드 넣어주세요.
```suggestion
            ItemsView = view ?? throw new ArgumentNullException(nameof(view));
            Container = view.ContentContainer;
```